### PR TITLE
Features

### PR DESCRIPTION
**Pull request by edwinmoradian90**
The only thing that I didn't do was get the icons in the nav because the links on apples page werent working for me for some reason.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2,27 +2,117 @@
...
-            <li>Store</li>
-            <li>Mac</li>
-            <li>iPod</li>
-            <li>iPhone</li>
-            <li>iPad</li>
-            <li>iTunes</li>
-            <li>Support</li>
-            <li>Search Icon</li>
+  <div class="container">
+    <main class="background">
```
Comment (stratospherique) [inline index.html]: - Semantically: the nav element shouldn't go inside a main element but a **header** element

**Comment by stratospherique** [comment]
### That's it. When finished fixing the above, kindly resubmit another code review :pray: . Keep up the good work :muscle: :muscle: .

**Comment by stratospherique** [comment]
> Thanks for review. I have some questions.
> 
> 1.) what do you mean occupy the full page? The apple site on my computer shows a container.
#### I used you html file to create a review, and based on that I did my review. As you can see the screenshot above :arrow_double_up: . Can you give us a screenshot of how it looks on your screen? Nevertheless, I'm not considering responsiveness, that you will be held accountable on in the next projects :nerd_face: .
> 2.) The apple site uses spans, is that okay or should I use literal buttons?
#### -  You are wrapping an `li` inside an anchor, it supposed to be the other way arround
> 3.) The four img elements are wrapped with a tags already, is there something im missing?
#### - They are not missing, but they are not event, the right most one is the biggest
> 4.) The footer has all those things that you mentioned unless im missing something,
- Yeah it exists, my mistake :pray:

File: index.html [hunk 2]
```diff
@@ -1,28 +1,147 @@
...
\ No newline at end of file
+  <body>
+    <div class="container">
+      <main class="background">
+        <div class="navigation-container flex aic">
+          <nav class="flex jcc">
+            <ul class="flex fdr jcc nb white">
+              <li>
+                <a href="#">
+                  <!--<img src="" alt="Apple Logo" />-->
```
Comment (samgaco) [inline index.html]: You should delete all the commented code in your project.

**Comment by edwinmoradian90** [comment]
apple-clone

**Comment by edwinmoradian90** [comment]
okay I see what you meant. Thanks for the suggestion

**Pull request merged**